.github/workflows/reusable-e2e.yml

```diff
@@ -323,6 +323,7 @@ jobs:
             packages/react-native
             packages/react-core
             packages/ui
+            packages/e2e
```

Comment:
Trigger e2e's when tests change, since features are shared btw cypress and detox test, this helps ensure changing features doesn't break RN, also useful to run tests when adding them :)